hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -570,6 +574,48 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 	return ctrl.Result{}, nil
 }
 
+func (r NodePoolReconciler) reconcileKubevirtMachineTemplate(ctx context.Context,
```

Comment:
Let's put this into its own kubevirt.go file in this package.

Reply:
Moved
Do you think have to keep this function under NodePoolReconciler struct?